### PR TITLE
custom 404 page for github pages

### PR DESCRIPTION
**Pull request by ericawright**
- add a custom 404 page that just redirects to the home page.
- unable to test as it is Github magic which will select the 404.html page and use it.

- in this case it might be nice to have a notification for the user that they have reach a 404 and have been redirected.

### CODE REVIEW

File: 404.html [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Firefox Design System</title>
+  </head>
+  <body>
+    <script type="text/javascript">
+      window.location.replace("https://bwinton.github.io/StyleGuide/");
```
Comment (bwinton) [inline 404.html]: Can we replace this with just `"/StyleGuide/"`, so that it'll work on the FirefoxUX github site, too?

File: 404.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Firefox Design System</title>
```
Comment (bwinton) [inline 404.html]: Also, we don't need JavaScript here.  We could use a [`<meta refresh>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta#Examples) tag instead, which I think would be cleaner.  🙂

Comment (ericawright) [inline 404.html]: I can do `window.location.origin + "/StyleGuide/"`

File: 404.html (hunk 1 above)
Comment (ericawright) [inline 404.html]: I can do `window.location.origin + "/StyleGuide/"`

Comment (bwinton) [inline 404.html]: You could, maybe, but I think the meta refresh would be easier, and should let you just use "/StyleGuide/".

Comment (ericawright) [inline 404.html]: No examples of meta refresh have just the pathname - but I'll try it

**Pull request merged**